src/gui/org/deidentifier/arx/gui/model/ModelStackelbergPrivacyCriterion.java

```diff
@@ -0,0 +1,122 @@
+package org.deidentifier.arx.gui.model;
+
+import org.deidentifier.arx.ARXStackelbergConfiguration;
+import org.deidentifier.arx.DataSubset;
+import org.deidentifier.arx.criteria.PrivacyCriterion;
+import org.deidentifier.arx.criteria.StackelbergPrivacyModel;
+import org.deidentifier.arx.gui.resources.Resources;
+import org.deidentifier.arx.gui.view.SWTUtil;
+
+public class ModelStackelbergPrivacyCriterion extends ModelImplicitCriterion {
+
+	private static final long serialVersionUID = -3632131196984677525L;
+	
+    private double             publisherBenefit  = 0d;
+    private double             publisherLoss     = 0d;
+    private double             adversaryGain     = 0d;
+    private double             adversaryCost     = 0d;
+
+	/** Parameter for the journalist model */
+    private DataSubset         subset            = null;

```

Comment:
DataSubset may not be necessary.  Included for continuity with framework representation.